package.json

```diff
@@ -1,11 +1,19 @@
 {
   "name": "express-gateway-plugin-example",
-  "version": "1.0.1",
+  "version": "1.1.0",
   "description": "Plugin Example for Express Gateway",
   "main": "manifest.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "",
-  "license": "Apache-2.0"
+  "license": "Apache-2.0",
+  "dependencies": {},
+  "devDependencies": {
+    "eslint": "^4.9.0",
+    "eslint-config-standard": "^10.2.1",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-node": "^5.2.1",
+    "eslint-plugin-promise": "^3.6.0"
```

Comment:
`eslint-plugin-node` and `eslint-plugin-promise` are being installed, but aren't imported in .eslintrc
You should add those to the `plugin` section of that file (note — `plugin-import` is required by `config-standard`)